### ISSUE TITLE
Create way to parameterize benchmarks

### ISSUE THREAD

**Issue by mdboom** [spacetelescope/asv#36]
It's probably handy to do:

```
def time_size(n):
    x = [0] * n
time_size.params = [10, 1000, 100000]
```

**Comment by cdeil** [comment]
+1 ... this would be nice!

**Comment by pv** [comment]
Just a heads up: some work towards this coming up here: https://github.com/pv/asv/tree/param-test
Running & js plot generation should work, x-axis and parameter set selection is still missing from UI but should be done soon-ish. Nevertheless, comments on [parameter specification API](https://github.com/pv/asv/blob/param-test/docs/source/writing_benchmarks.rst#parameterized-benchmarks) could be useful already at this stage  --- I'm just basing this on what would be useful for Scipy benchmarks, and experiences may vary...

**Comment by pv** [comment]
This feature should be there now.

**Comment by mdboom** [comment]
Fixed by #205.
